/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/torndb.py:234: Warning: Table 'log2016_10_22' already exists
  return cursor.execute(query, kwparameters or parameters)
/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/torndb.py:234: Warning: Table 'log2016_10_22' already exists
  return cursor.execute(query, kwparameters or parameters)
Traceback (most recent call last):
  File "/home/v-wxb-chai/workspace/webserver/SEMain/app.py", line 438, in <module>
    http_server.listen(options.port)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/tornado/tcpserver.py", line 126, in listen
    sockets = bind_sockets(port, address=address)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/tornado/netutil.py", line 196, in bind_sockets
    sock.bind(sockaddr)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Traceback (most recent call last):
  File "/home/v-wxb-chai/workspace/webserver/SEMain/app.py", line 438, in <module>
    http_server.listen(options.port)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/tornado/tcpserver.py", line 126, in listen
    sockets = bind_sockets(port, address=address)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/tornado/netutil.py", line 196, in bind_sockets
    sock.bind(sockaddr)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Traceback (most recent call last):
  File "/home/v-wxb-chai/workspace/webserver/SEMain/app.py", line 438, in <module>
    http_server.listen(options.port)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/tornado/tcpserver.py", line 126, in listen
    sockets = bind_sockets(port, address=address)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/tornado/netutil.py", line 196, in bind_sockets
    sock.bind(sockaddr)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Traceback (most recent call last):
  File "/home/v-wxb-chai/workspace/webserver/SEMain/app.py", line 438, in <module>
    http_server.listen(options.port)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/tornado/tcpserver.py", line 126, in listen
    sockets = bind_sockets(port, address=address)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/site-packages/tornado/netutil.py", line 196, in bind_sockets
    sock.bind(sockaddr)
  File "/home/v-wxb-chai/anaconda2/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
[W 161022 02:38:52 web:1946] 405 CONNECT www.baidu.com:443 (14.17.37.19) 1.72ms
[W 161022 02:38:52 web:1946] 404 GET http://www.baidu.com/ (14.17.37.19) 0.40ms
[I 161022 03:24:37 web:1946] 200 GET / (167.114.30.100) 3.81ms
[W 161022 08:24:50 web:1946] 404 GET http://www.izhongsai.com/ (112.125.92.238) 1.19ms
[I 161022 12:38:52 web:1946] 200 GET /queryAll?q=%E7%82%8E%E9%BB%84%E6%98%A5%E7%A7%8B%20%E6%9D%9C%E5%AF%BC%E6%AD%A3%20%E6%96%A9%E9%A6%96&baidu=true&s360=true&sogou=true&content_type=2&pn=1 (106.121.76.141) 10.61ms
[W 161022 12:38:54 web:1946] 404 GET /apple-touch-icon-120x120-precomposed.png (106.121.76.141) 1.13ms
[W 161022 12:38:54 web:1946] 404 GET /apple-touch-icon-120x120.png (106.121.76.141) 0.57ms
[W 161022 12:38:54 web:1946] 404 GET /apple-touch-icon-precomposed.png (106.121.76.141) 0.57ms
[W 161022 12:38:54 web:1946] 404 GET /apple-touch-icon.png (106.121.76.141) 0.52ms
[W 161022 12:38:54 web:1946] 404 GET /favicon.ico (106.121.76.141) 0.85ms
[W 161022 12:38:54 web:1946] 404 GET /apple-touch-icon-120x120-precomposed.png (106.121.76.141) 0.55ms
[W 161022 12:38:55 web:1946] 404 GET /apple-touch-icon-120x120.png (106.121.76.141) 0.50ms
[W 161022 12:38:55 web:1946] 404 GET /apple-touch-icon-precomposed.png (106.121.76.141) 0.50ms
[W 161022 12:38:55 web:1946] 404 GET /apple-touch-icon.png (106.121.76.141) 0.52ms
[W 161022 12:38:55 web:1946] 404 GET /favicon.ico (106.121.76.141) 0.82ms
[W 161022 14:15:10 web:1946] 404 GET /favicon.ico (106.120.0.181) 0.80ms
[I 161022 14:17:01 web:1946] 200 GET /queryAll?q=%E7%82%8E%E9%BB%84%E6%98%A5%E7%A7%8B%20%E6%9D%9C%E5%AF%BC%E6%AD%A3%20%E6%96%A9%E9%A6%96&baidu=true&s360=true&sogou=true&content_type=2&pn=1 (202.43.148.166) 10.12ms
[W 161022 14:17:01 web:1946] 404 GET /apple-touch-icon-120x120-precomposed.png (202.43.148.166) 1.19ms
[W 161022 14:17:01 web:1946] 404 GET /apple-touch-icon-120x120.png (202.43.148.166) 0.54ms
[W 161022 14:17:01 web:1946] 404 GET /apple-touch-icon-precomposed.png (202.43.148.166) 0.82ms
[W 161022 14:17:01 web:1946] 404 GET /apple-touch-icon.png (202.43.148.166) 0.54ms
[W 161022 14:17:01 web:1946] 404 GET /favicon.ico (202.43.148.166) 1.05ms
[W 161022 14:17:01 web:1946] 404 GET /apple-touch-icon-120x120-precomposed.png (202.43.148.166) 0.38ms
[W 161022 14:17:01 web:1946] 404 GET /apple-touch-icon-120x120.png (202.43.148.166) 0.82ms
[W 161022 14:17:01 web:1946] 404 GET /apple-touch-icon-precomposed.png (202.43.148.166) 0.70ms
[W 161022 14:17:01 web:1946] 404 GET /apple-touch-icon.png (202.43.148.166) 0.63ms
[W 161022 14:17:01 web:1946] 404 GET /favicon.ico (202.43.148.166) 0.69ms
[W 161022 15:15:14 web:1946] 404 GET http://www.baidu.com/cache/global/img/gs.gif (80.82.78.38) 0.55ms
